api/mapping/models.py

```diff
@@ -499,32 +510,31 @@ class StructuralMappingRule(BaseModel):
     """
     To come
     """
-
+    #save the scan_report link to make it easier
     scan_report = models.ForeignKey(ScanReport, on_delete=models.CASCADE)
 
     omop_field = models.ForeignKey(OmopField, on_delete=models.CASCADE)
 
-    source_table = models.ForeignKey(
```

Comment:
Can we keep this fields for now, and add a TODO to stop using it. It will simplify testing this code without much changes in the models.